teammapper-frontend/mmp/src/map/handlers/nodes.ts

```diff
@@ -889,6 +943,24 @@ export default class Nodes {
         }
     }
 
+    /**
+     * Update the node hidden value
+     * @param {Node} node
+     * @param {boolean} hidden
+     * @returns {boolean}
+     */
+    private updateNodeHidden = (node: Node, hidden: boolean) => {
+        if (hidden && typeof hidden !== 'boolean') {
+            Log.error('The hidden value must be boolean', 'type')
+        }
+
+        if (node.hidden !== hidden) {
+            node.hidden = hidden;
```

Comment:
please check to return a consistent return type. here, you either return false or you assign a variable and return undefined (which is also false).

Reply:
All the other methods also either assign a variable or return false, so I was just orienting myself on those. Maybe a general refactoring idea?

Reply:
imho only because the other methods follow a bad practice, this doesn't mean we have to continue this way. The return value is useless if it's always false? But yes, the other methods could be refactored in a separate refactoring.

Reply:
Yeah, I've refactored this one to return undefined, I'll open an issue for the other ones.